maps/atlas.dmm

```diff
@@ -17792,6 +17933,22 @@
 	},
 /turf/simulated/floor/black,
 /area/station/crew_quarters/bar)
+"nBe" = (
+/obj/item/parts/robot_parts/arm/left/standard,
+/obj/item/parts/robot_parts/arm/right/standard,
+/obj/item/parts/robot_parts/leg/left/standard,
+/obj/item/parts/robot_parts/leg/right/standard,
+/obj/item/parts/robot_parts/leg/left/treads,
+/obj/item/parts/robot_parts/leg/right/treads,
+/obj/storage/crate/medical{
+	name = "medical supplies crate"
+	},
+/obj/item/storage/firstaid/brain,
+/obj/item/storage/firstaid/brute,
+/obj/item/storage/firstaid/fire,
+/obj/item/storage/firstaid/oxygen,
```

Comment:
Again, this is a lot more medical supplies than any other listening post has (afaik). Try to keep them relatively standardized.